node/mdSchema.ts

```diff
@@ -19,6 +19,8 @@ export const ORGANIZATION_FIELDS = [
   'collections',
   'paymentTerms',
   'priceTables',
+  'organizationCustomFields',
+  'costCenterCustomFields',
```

Comment:
Again, should just be one `customFields` field here.

Reply:
✅